android_app/app/src/main/res/values/strings.xml

```diff
@@ -70,55 +70,55 @@
     <string name="label_export">Export</string>
     <string name="label_delete_all">Delete all</string>
 
-    <string name="error_value_required">Value is required</string>
-    <string name="error_value_range">The value is not in range</string>
+    <string name="error_value_required">Value required</string>
+    <string name="error_value_range">The value not in range</string>
     <string name="error_exporting">Error exporting</string>
     <string name="error_importing">Error importing</string>
-    <string name="error_user_name_required">Error: user name is required</string>
-    <string name="error_user_name_too_short">Error: user name have to be at least 3 characters</string>
-    <string name="error_height_required">Error: height is required</string>
-    <string name="error_initial_weight_required">Error: initial weight is required</string>
-    <string name="error_goal_weight_required">Error: goal weight is required</string>
-
-    <string name="info_data_deleted">Database entry deleted</string>
-    <string name="info_data_all_deleted">All database entries deleted</string>
-    <string name="info_data_exported">Data exported to</string>
-    <string name="info_data_imported">Data imported from</string>
-    <string name="info_enter_value_cm">Enter value in cm</string>
-    <string name="info_enter_comment">Enter an optional comment</string>
-    <string name="info_enter_value_scale_unit">Enter value in scale unit</string>
+    <string name="error_user_name_required">Error: Username required</string>
+    <string name="error_user_name_too_short">Error: Username must be 3 characters or more</string>
+    <string name="error_height_required">Error: Height required</string>
+    <string name="error_initial_weight_required">Error: Initial weight required</string>
+    <string name="error_goal_weight_required">Error: Goal weight required</string>
+
+    <string name="info_data_deleted">Entry deleted</string>
+    <string name="info_data_all_deleted">All entries deleted</string>
+    <string name="info_data_exported">Exported to</string>
+    <string name="info_data_imported">Imported from</string>
+    <string name="info_enter_value_cm">Value in cm:</string>
```

Comment:
This and the two below should not end with colon. They are shown inside a text edit where I think it looks weird with colon.

Reply:
Fine by me